aasemble/django/apps/buildsvc/models.py

```diff
@@ -328,6 +328,7 @@ def long_name(self):
     def name(self):
         return self.git_url.split('/')[-1].replace('_', '-')
 
+    @property
     def build(self):
```

Comment:
@pljensen Any suggestions on recommended function call to use to trigger a rebuild?